README.md

```diff
@@ -23,6 +23,7 @@
 | PROJECTNAME          | Name of the Project.                              | NONE * |
 | VERSION              | Version of the Project.                           | NONE * |
 | LOGO                 | An image to be used as logo for the Project.      | path relative to DOCPATH. *example* - <i>To use DOCPATH/images/logo.svg as the logo, set LOGO as images/logo.svg</i>.|
+| MARKDOWN_FLAVOUR     | Input file format flavour. (You can see list of supported format [here](http://pandoc.org/MANUAL.html))                         | markdown_github   |
```

Comment:
https://github.com/jgm/pandoc/blob/4bae307616de5688983b55cbec1819c2e2864c6e/src/Text/Pandoc/Readers.hs#L107 states that pandoc supports 6 flavours of markdown
- markdown
- markdown_strict
- markdown_phpextra
- markdown_github
- markdown_mmd
- commonmark